scripts/install-mender.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
+
+set -e
+
+CHANNEL="stable"
+
+DEFAULT_COMPONENTS="mender-client"
+COMPONENTS="$DEFAULT_COMPONENTS mender-shell"
+SELECTED_COMPONENTS=$DEFAULT_COMPONENTS
+
+export DEBIAN_FRONTEND=noninteractive
+
+banner (){
+    echo "
+                          _
+ _ __ ___   ___ _ __   __| | ___ _ __ 
+| '_ \` _ \ / _ \ '_ \ / _\` |/ _ \ '__|
+| | | | | |  __/ | | | (_| |  __/ |   
+|_| |_| |_|\___|_| |_|\__,_|\___|_|   
+
+Running the Mender installation script.
+--
+"
+                                      
+}
+
+usage() {
+    echo "usage: install-mender.sh [-h] [-c channel] [component...]"
+    echo "  -h             print this help"
+    echo "  -c             channel: stable(default)|experimental"
+    echo "  <component>    list of components to install"
+    echo ""
+    echo "Supported components:"
+    echo "  * mender-client (default if none provided)"
+    echo "  * mender-shell"
```

Comment:
I also suggest we generate this list based on a defined variable above to ease maintenance. (As I commented, by default should be all, so that we can print before or after, and then just generate a list based on a variable).